src/components/GameView/GameOverlays.vue

```diff
@@ -5,42 +5,54 @@
       v-model="waitingForOpponentToCounter"
       class="game-overlay"
     >
-      <h1 class="text-h3">{{ showWaitingForOpponetToCounterMessage }}</h1>
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+        {{ showWaitingForOpponetToCounterMessage }}
+      </h1>
     </v-overlay>
     <v-overlay
       id="waiting-for-opponent-discard-scrim"
       v-model="waitingForOpponentToDiscard"
       class="game-overlay"
     >
-      <h1 class="text-h3">Opponent Is Discarding</h1>
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+        Opponent Is Discarding
+      </h1>
     </v-overlay>
     <v-overlay
       id="waiting-for-opponent-resolve-three-scrim"
       v-model="waitingForOpponentToPickFromScrap"
       class="game-overlay"
     >
-      <h1 class="text-h3">Opponent Choosing Card from Scrap</h1>
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+        Opponent Choosing Card from Scrap
+      </h1>
     </v-overlay>
     <v-overlay
       id="waiting-for-opponent-play-from-deck-scrim"
       v-model="showWaitingForOpponentToPlayFromDeck"
       class="game-overlay"
     >
-      <h1 class="text-h3">Opponent Playing from Deck</h1>
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+        Opponent Playing from Deck
+      </h1>
     </v-overlay>
     <v-overlay
       id="waiting-for-opponent-to-discard-jack-from-deck"
       v-model="showWaitingForOpponentToDiscardJackFromDeck"
       class="game-overlay"
     >
-      <h1 class="text-h3">Opponent Must Discard Jack</h1>
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+        Opponent Must Discard Jack
+      </h1>
     </v-overlay>
     <v-overlay
       id="waiting-for-opponent-stalemate-scrim"
       v-model="waitingForOpponentToStalemate"
       class="game-overlay"
     >
-      <h1 class="text-h3">Opponent Considering Stalemate Request</h1>
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3']">
+        <div>Opponent Considering Stalemate Request</div>
+      </h1>
```

Comment:
These are all much better, however, I have a separate problem with these all being H1s. Accessibiity states that each page should only have ONE primary heading. These should all probably be h2s I'd guess. Leave that for another ticket though since they were h1s already.

The classes themselves look good! 👍🏼

Reply:
I'm not as strong on accessibility so @seriouslysean penny for your thoughts: does the page only rendering one of these (at most, exclusively) reduce the badness? The <template> code does have more than 1 `<h1>` but the rendered html has only 1 at a time

Reply:
Only if this is the only h1 on the page. I’m fine leaving it and eventually doing an accessibility pass.